service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftCMOControllerTest.java

```diff
@@ -44,21 +44,17 @@
 class DraftCMOControllerTest {
     private static final String AUTH_TOKEN = "Bearer token";
     private static final String USER_ID = "1";
-
+    private final LocalDateTime date = LocalDateTime.now();
```

Comment:
given we are here already, could we rename appropriately? Probable `todaysDate`?